waiter/src/waiter/scheduler.clj

```diff
@@ -555,12 +562,10 @@
 
                        state-query-chan
```

Comment:
Do we still need a `state-query-chan` now that there is a `retrieve-syncer-state-fn`?